app/Exports/CirclesExport.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Exports;
+
+use App\Eloquents\Circle;
+use Maatwebsite\Excel\Concerns\FromCollection;
+use Maatwebsite\Excel\Concerns\WithHeadings;
+use Maatwebsite\Excel\Concerns\WithMapping;
+
+class CirclesExport implements FromCollection, WithHeadings, WithMapping
+{
+    private $headings = [
+        'id' => 'id',
+        'name' => '企画名',
+        'name_yomi' => '企画名（よみ）',
+        'group_name' => '団体名',
+        'group_name_yomi' => '団体名（よみ）',
+        'status' => 'ステータス',
+        'notes' => 'スタッフメモ',
+        'leader_id' => '責任者 ユーザーID',
+        'leader_student_id' => '責任者 学籍番号',
+        'leader_name' => '責任者 氏名',
+    ];
+
+    /**
+    * @return \Illuminate\Support\Collection
+    */
+    public function collection()
+    {
+        return Circle::Submitted()->with('leader')->get();
```

Comment:
スコープ名称はいままで小文字で統一してきたので、ここでも小文字にしよう

```suggestion
        return Circle::submitted()->with('leader')->get();
```